homeassistant/components/garages_amsterdam/config_flow.py

```diff
@@ -0,0 +1,44 @@
+"""Config flow for Garages Amsterdam integration."""
+import garages_amsterdam
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Garages Amsterdam."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    _options = None
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        errors = {}
+
+        if self._options is None:
+            self._options = []
+            api_data = await garages_amsterdam.get_garages(
```

Comment:
We need to have a try... except block here and catch some aiohttp error and notify the user of the connection error.